dashboard/src2/pages/LoginSignup.vue

```diff
@@ -77,38 +77,104 @@
 									v-model="email"
 									required
 								/>
```

Comment:
This file is too big. Maybe consider refactoring it into smaller components. Maybe later.

Reply:
Agreed, will do.